ratelimits/names.go

```diff
@@ -39,18 +41,25 @@ const (
 	NewOrdersPerAccount
 
 	// FailedAuthorizationsPerAccount uses bucket key 'enum:regId', where regId
-	// is the registration id of the account.
+	// is the ACME registration Id of the account.
 	FailedAuthorizationsPerAccount
 
-	// CertificatesPerDomainPerAccount uses bucket key 'enum:regId:domain',
-	// where name is the a name in a certificate issued to the account matching
-	// regId.
+	// CertificatesPerDomain uses bucket key 'enum:domain', where domain is a
+	// domain name in the issued certificate.
+	CertificatesPerDomain
+
```

Comment:
It's important to note that this kind of change -- inserting a new entry in the _middle_ of the enum -- can never happen once we start actually persisting this data anywhere. It's a backwards-incompatible change like changing the number of a protobuf field.

Reply:
Absolutely.